server/utils/import_contacts.coffee

```diff
@@ -140,19 +140,21 @@ module.exports = (token, callback) ->
         , (err)->
             return callback err if err
             async.eachSeries contacts.google, (gContact, cb) ->
-                if updatedContacts[gContact.id.$t]?
-                    addContactPicture updatedContacts[gContact.id.$t], gContact, (err) ->
-                        log.debug "picture err #{err}"
+                updated = updatedContacts[gContact.id.$t]
+                if updated?
+                    addContactPicture updated, gContact, (err) ->
+                        log.debug "picture err #{err}" if err
                         setTimeout cb, 10
                 else
                     cb()
             , (err) ->
                 return callback err if err
 
+                _ = localizationManager.t
                 notification.createOrUpdatePersistent "leave-google-contacts",
```

Comment:
Maybe be careful with the use of `_` varname, because it's already mapped to _underscore.js_ at global scope. No incidence here, but keep it in mind :wink: